my-react-app/src/components/Form/Form.tsx

```diff
@@ -0,0 +1,152 @@
+import PhoneInput from "react-phone-number-input/input";
+import { useForm } from "react-hook-form";
+import "react-phone-number-input/style.css";
+/* import CountryList from "./CountryList"; */
+import { useState } from "react";
+import clsx from "clsx";
+import Input from "./Input";
+
+interface FormData {
+  firstName: string;
+  lastName: string;
+  title?: string;
+  street?: string;
+  city?: string;
+  zip: string;
+  country: string;
+  phoneNumber?: string;
+}
+
+const Form = () => {
+  const [check, setCheck] = useState(false);
+  const [scale, setScale] = useState(false);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isValid },
+    setValue,
+    getValues,
+  } = useForm<FormData>();
+
+  const onSubmit = (data: FormData) => {
+    console.log(data);
+    if (isValid) {
+      setCheck(true);
```

Comment:
was machst da ?

Reply:
das ist für die Submit Successful message, damit es so aufscheint und 4 sekunden später verschwunden wird, vielleicht ist das nicht die richtige variante für Animation in react, wollte aber ein bisschen mit setTimeout spielen

Reply:
ah okay dann alles richtig gemacht :)